### DISCUSSION TITLE
Logo proposal

### DISCUSSION THREAD

**Discussion by mnesarco** [terjeio/grblHAL#268] [Ideas]
Hi Friends,
I have this logo proposal:

![image](https://user-images.githubusercontent.com/1087317/109226054-24b68400-778c-11eb-9f77-27b1690af31d.png)

**Comment by phil-barrett** [comment]
I have a number of grblHAL treatments. Not married to any of them, just needed something to use. The last one is an animated gif but I'm not terribly happy with it.  I have been using these on grbl.org for some time now.

The original grbl logo is black so I think we should stick to that color to emphasize grblHAL's roots. I used red for HAL because, well, I like red and black and red go well together. I'm open to other than red, though. Note, I do not know if we need to get clearance/licensing to use the grbl logo. I think it falls within the open source license but I am not a lawyer.

![grblhal](https://user-images.githubusercontent.com/16570958/109233384-a99aa100-777e-11eb-99ed-d46a13fee736.png)
![grblhal driven](https://user-images.githubusercontent.com/16570958/109233412-b3240900-777e-11eb-8709-e0795fc101c8.png)
![grbl-to-grblhal fade](https://user-images.githubusercontent.com/16570958/109233423-b6b79000-777e-11eb-91bf-2ed3dbab5559.gif)

**Comment by mnesarco** [comment]
Ok, this is a Black&Red version of the proposal:

![image](https://user-images.githubusercontent.com/1087317/109250410-a5d74080-77b7-11eb-9c7e-2791915cd66c.png)

**Reply by Klippert** [reply]
Cool - i think this one would be my favourite so far. it's something that should cut/engrave nicely ;)

**Comment by mnesarco** [comment]
The usual problem with Black&Red Logos is that they don't play well with dark backgrounds.

Some colors play better with light and dark backgrounds:

![image](https://user-images.githubusercontent.com/1087317/109250994-a9b79280-77b8-11eb-8f49-3eec21856f58.png)

**Comment by Klippert** [comment]
Did this thread just die? Or was there other opinions? Maybe a poll is in order?

**Reply by phil-barrett** [reply]
I think it is up to Terje on what he wants to do.  I will continue use the logos I did until there is clarity. At that point I will probably go with what ever is decided on.  

There may need to be a little bit of legalese - open source license or some such.

**Reply by Klippert** [reply]
Okie. It's nice to see people contribute though.

**Reply by terjeio** [reply]
Shall we wait a bit before deciding, maybe other designs will emerge in due time?

I have no idea of a design of my own, and I want to leave it to you to decide on which to choose. So up to you (if agreement can be reached).

**Reply by mnesarco** [reply]
Hi Friends,

I know this is a non technically critical discussion, but it seems that nobody else is interested in proposing more logo options. As this whole project is a reborn of grbl, I feel that it is the moment to consolidate a strong "branding" image. grbl.org needs a major redesign, right now it is very confusing site, surprisingly there is no link from there to the github repositories. Many people may think that grbl is dead along with 8bit boards.
